runtime/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/config/MqttBindingConfig.java

```diff
@@ -206,4 +246,18 @@ private static Function<String, String> orElseIfNull(
             return result != null ? result : second.apply(x);
         };
     }
+
+    private static class TopicValidator
+    {
+        ModelConfig content;
+        Map<Matcher, ModelConfig> userProperties;
```

Comment:
Why is this key in the map a `Matcher`?
If matcher is not needed, then we can use `MqttTopicConfig` instead of `TopicValidator`, right?
If we need to keep this to key by `String16FW` in the user properties map, then we can name this `MqttTopicModel` instead.

Reply:
Topics contain the user properties, and when a publish comes, we first need to find the validator for that topic. E.g: the configured pattern is `sensors/#` and we publish to `sensors/1`, we need to match against the pattern to find the first topic validator, which contains the user property validator.